src/containers/ViewerPanel/index.tsx

```diff
@@ -15,24 +15,30 @@ import Bowser from "bowser";
 
 import { State } from "../../state/types";
 import selectionStateBranch from "../../state/selection";
-import metadataStateBranch from "../../state/metadata";
-import { VIEWER_EMPTY, VIEWER_SUCCESS } from "../../state/metadata/constants";
+import trajectoryStateBranch from "../../state/trajectory";
+import viewerStateBranch from "../../state/viewer";
+import {
+    VIEWER_EMPTY,
+    VIEWER_SUCCESS,
+    VIEWER_ERROR,
```

Comment:
was ERROR added intentionally?  didn't see it referenced yet

Reply:
It was moved up from line 35 in the "before" version :)

Reply:
I see, thanks!